cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -780,7 +780,9 @@ function Configure-HostNetworkingService {
     netsh interface ipv4 set interface "${vnic_name}" forwarding=enabled
   }
 
-  Get-HNSPolicyList | Remove-HnsPolicyList
+  Try {
+    Get-HNSPolicyList | Remove-HnsPolicyList
+  } Catch { }
```

Comment:
Does the exception get printed even if it's caught? If not, can you add a line to print it? (I think there should be an example elsewhere in this file).